src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -22,6 +23,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Target Name="_ComputeTargetFrameworkItems" Returns="@(InnerOutput)">
     <ItemGroup>
       <_TargetFramework Include="$(TargetFrameworks)" />
```

Comment:
I don't think so, the scope of the `_TargetFramework ` since to be local to the `_ComputeTargetFrameworkItems` target. For now :)